Assets/Minigames/FrequencyMinigame/Sinewave.cs

```diff
@@ -6,24 +6,22 @@ public class Sinewave : MonoBehaviour
 {
     public LineRenderer lineRenderer;
     public int numberPoints;
-    public float amplitude = 1.75f;
-    public float frequency = 0.2f;
+    public float amplitude = 1.75f; // please make sure that the value is in the amplitude limit [0.5 , 3]
```

Comment:
I think instead of adding a comment, we should add a `Range` decorator. Example:
```cs
[Range(0.5f, 3f)]
public float amplitude = 1.75;
```